src/libraries/System.ComponentModel.Annotations/src/System/ComponentModel/DataAnnotations/ValidationResult.cs

```diff
@@ -23,7 +23,7 @@ public class ValidationResult
         ///     The <c>null</c> value is used to indicate success.  Consumers of <see cref="ValidationResult" />s
         ///     should compare the values to <see cref="ValidationResult.Success" /> rather than checking for null.
         /// </remarks>
-        public static readonly ValidationResult Success;
+        public static readonly ValidationResult? Success;
```

Comment:
This still makes me chuckle--I remember coming to the conclusion we needed this field even though it would always be null. Too bad there isn't a way to annotate "yeah, this is _always_ gonna be null." :wink:

Reply:
Yeah :) In general I've not yet annotated other APIs where null means success, it's a bit odd.

Reply:
Indeed it is -- it might be the only place across our libraries.  We knew consumers of validation results needed to easily compare against `Success`, but we didn't want to have an `IsValid` property on the result that folks would have to check either. We went a while without `ValidationResult.Success` and tried writing application code just using `null` but it wasn't very expressive. So we decided to create `ValidationResult.Success` as an expressive way to say `null`.